### ISSUE TITLE
snappyHexMesh GUI add-on in MacBook Pro M1 Ventura 

### ISSUE THREAD

**Issue by ClaudioCor** [tkeskita/snappyhexmesh_gui#15]
I need help, because I couldn't install snappyHexMesh GUI add-on in my Mac. After downloading this add-on and start the blender and  follow the tutorial "SnappyHexMesh GUI Add-on for Blender Documentation, release 0.9", by Tuomo Keskilato,  I couldn't install it. The add-on is not visible in sidebar after loading a file.stl. In my linux notebook I had no  problem for that.
I have a MacBook Pro M1 with macOS Ventura 13.3.1. 
Regards.

**Comment by tkeskita** [comment]
Hi,
Did you get some error message during add-on installation? If you [start Blender from terminal](https://docs.blender.org/manual/en/latest/advanced/command_line/launch/macos.html) it may show some extra error messages in the terminal. Do you see the add-on listed in Blender preferences?

**Comment by tkeskita** [comment]
Hi,
there seems to be something wrong in your addons folder. The `.../Blender/x.y/scripts/addons` folder should contain add-on sources in separate subfolders. Can you try to empty that addons folder, start blender again and install again?

**Comment by ClaudioCor** [comment]
When downloading the add-on folder in zip format, my Mac unzipped the folder, I don't know why. So, I zip the folder again and start the Blender. I try to install add-on again and new errors appears:

Read prefs: /Users/claudiocorrea/Library/Application Support/Blender/3.5/config/userpref.blend
Device with name Apple M1 supports metal minimum requirements
METAL API - DETECTED GPU: Apple M1
fake_module: addon missing 'bl_info' gives bad performance!: '/Users/claudiocorrea/Library/Application Support/Blender/3.5/scripts/addons/blockBuilder.py'
fake_module: addon missing 'bl_info' gives bad performance!: '/Users/claudiocorrea/Library/Application Support/Blender/3.5/scripts/addons/conf.py'
fake_module: addon missing 'bl_info' gives bad performance!: '/Users/claudiocorrea/Library/Application Support/Blender/3.5/scripts/addons/blockBuilder.py'
fake_module: addon missing 'bl_info' gives bad performance!: '/Users/claudiocorrea/Library/Application Support/Blender/3.5/scripts/addons/conf.py'
fake_module: addon missing 'bl_info' gives bad performance!: '/Users/claudiocorrea/Library/Application Support/Blender/3.5/scripts/addons/blockBuilder.py'
fake_module: addon missing 'bl_info' gives bad performance!: '/Users/claudiocorrea/Library/Application Support/Blender/3.5/scripts/addons/conf.py'
reloading addon: snappyhexmesh_gui-master.zip 1681737834.3130703 1681737997.639257 '/Users/claudiocorrea/Library/Application Support/Blender/3.5/scripts/addons/snappyhexmesh_gui-master.zip/__init__.py'
Modules Installed () from '/Users/claudiocorrea/Downloads/snappyhexmesh_gui-master.zip' into '/Users/claudiocorrea/Library/Application Support/Blender/3.5/scripts/addons'
Traceback (most recent call last):
  File "/Applications/Blender.app/Contents/Resources/3.5/scripts/modules/addon_utils.py", line 333, in enable
    mod = __import__(module_name)
ModuleNotFoundError: No module named 'snappyhexmesh_gui-master'

Now I see the add-on in the preferences/community but I couldn't activate.

**Comment by ClaudioCor** [comment]
Yes, but I don't know how to proceed to locate this folder in Mac.

**Comment by tkeskita** [comment]
I don't have a Mac to test this, but try this in terminal: `ls /Users/yourusername/Library/Application\ Support/Blender/3.5/scripts/addons`

**Comment by ClaudioCor** [comment]
The files in addons: 
__MACOSX			conf.py
__init__.py			snappyhexmesh_gui-master.zip
blockBuilder.py

**Comment by tkeskita** [comment]
OK, try to delete files in the addons folder:
```
rm /Users/yourusername/Library/Application\ Support/Blender/3.5/scripts/addons/conf.py
rm /Users/yourusername/Library/Application\ Support/Blender/3.5/scripts/addons/init.py
rm /Users/yourusername/Library/Application\ Support/Blender/3.5/scripts/addons/snappyhexmesh_gui-master.zip
rm /Users/yourusername/Library/Application\ Support/Blender/3.5/scripts/addons/blockBuilder.py
```
then restart blender and try to reinstall add-on.

P.S. You use Blender version 3.5, but 3.3 LTS is supported. However, it probably works on 3.5, but I haven't tried.

**Comment by ClaudioCor** [comment]
Hi, I fixed the problem, uninstall 3.5 and install 3.3 LTS.
Thanks.
